### ISSUE TITLE
Hours widget integration

### ISSUE THREAD

**Issue by Jessdwilson** [ObjectiveSubject/ccl#32]

**Comment by Jessdwilson** [comment]
not sure if this is the same, Gabriel requested:

There are a couple SpringShare widgets that we'd like to include in the content body area. Specifically, our appointment module widget, and the calendar widget. They probably won't work unless we enqueue the scripts in the backend. Do you think we can do that?

For the appointment widget: `//api3.libcal.com/js/myscheduler.min.js?002`
for the calendar widget: `//api3.libcal.com/js/hours_month.js?002`

There might be more... but these are definitely important ones for starters

**Comment by kpettinga** [comment]
Waiting on Gabriel to provide us with more information about how these widgets will be integrated. 

> @gabriel_ortiz, Do you already have the code to paste for these widgets? Or is something you need us to create?

**Comment by davekellam** [comment]
I added the scheduler widget in sprint 2 (#15), as there was no way to deal with booking librarians via API. It can be included via the `[scheduler]` shortcode.

I poked around in the LibCal backend and found the code for the `hours_widget` mentioned. I've included that as `[calendar_widget]` shortcode. The `calendar_widget()` function in `/includes/integrations/libcal.php` includes the default generated css with minor tweaks (I removed rounded corners and used different grays for the different calendars).
